### PR TITLE
betty: add page

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/betty.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# betty
+
+> Translate phrases into commands.
+> More information: <https://github.com/pickhardt/betty>
```
Comment (owenvoke) [inline pages/common/betty.md:4]: This should fix the linter issues:
```suggestion
> More information: <https://github.com/pickhardt/betty>.
```

File: pages/common/betty.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`betty download {{https://example.com/something.png}} to something.png`
+
+- Compress something to whatever format you want:
+
+`betty compress {{path/to/dir}}`
+
+- Uncompress something:
+
+`betty uncompress {{archive.tar.gz}}`
```
Comment (bl-ue) [inline pages/common/betty.md:24]: Same here.
```suggestion
`betty {{unzip}} {{archive.tar.gz}}`
```

File: pages/common/betty.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Unarchive something to directory:
```
Comment (bl-ue) [inline pages/common/betty.md:26]:
```suggestion
- Extract an archive into a directory:
```

File: pages/common/betty.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`betty unarchive {{archive.tar.gz}} to {{directory}}`
+
+- Play Spotify:
+
+`betty play Spotify`
```
Comment (bl-ue) [inline pages/common/betty.md:32]:
```suggestion
`betty play {{Spotify}}`
```

File: pages/common/betty.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Drive Betty to madness
```
Comment (bl-ue) [inline pages/common/betty.md:34]:
```suggestion
- Drive Betty to madness:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Cool!

File: pages/common/betty.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# betty
+
+> Translate phrases into commands.
+> More information: <https://github.com/pickhardt/betty>
+
+- Ask something to Betty:
+
+`betty what time is it`
```
[the remark this answers is not in the document]
Comment (Yutyo) [reply pages/common/betty.md:8]: @bl-ue its not a section you change. Its exactly what you write in order to ask Betty what time is it in that moment.

[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/common/betty.md:8]: 👍🏻, but you can also ask other things, and the example description is 'Ask something'

[the remark this answers is not in the document]
Comment (Yutyo) [reply pages/common/betty.md:8]: Oh OK

**Comment by Yutyo** [comment]
All done?

File: pages/common/betty.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# betty
+
+> Translate phrases into commands.
```
Comment (bl-ue) [inline pages/common/betty.md:3]: How about something slightly clearer?
```suggestion
> Use natural language to execute commands.
```

**Review by bl-ue — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

**Comment by bl-ue** [comment]
Oops! I deleted your branch @Yutyo! :smile:

I'm using to doing that on my own and fellow maintainers' PRs&mdash;normally, though, we can't delete branches on users' forks.  I can, though, because you added me as a collaborator to your fork. 😀 

I'll take care of the merge commits for you, in a few moments.

**Pull request merged**